bowling/src/Player.cpp

```diff
@@ -0,0 +1,36 @@
+#include "../include/Player.hpp"
+#include <iostream>
+
+int Player::getScore() {
+    int score = 0;
+    std::vector<std::pair<int, int>> twoThrows;
+
+    for (auto i = 0; i < points_.size() - 1; i += 2) {
+        twoThrows.push_back(std::make_pair(points_[i], points_[i + 1]));
+    }
+    int extraBall = twoThrows.size();
+    int throwBalls = 9 < twoThrows.size() ? 9 : twoThrows.size();
```

Comment:
```suggestion
    int throwBalls = (9 < twoThrows.size() ? 9 : twoThrows.size());
```
add parentheses for clarity. Or use std::max/min